app/src/main/java/buzz/getcoco/iot/sample/adapters/ResourceTileAdapter.java

```diff
@@ -122,9 +122,14 @@ private void bind(ResourceEx resource, RecyclerItemResourceBinding binding, Life
             setControlValues(binding, (Boolean) currentValue);
           }
         });
+
+        binding.tvPowerDescription.setVisibility(View.VISIBLE);
+        binding.tvPowerValue.setVisibility(View.VISIBLE);
+        binding.btOnOff.setVisibility(View.VISIBLE);
```

Comment:
shouldn't these be moved into their corresponding observe methods of the livedata ?

Reply:
Moved to attribute value observable.